Firebase/ios/Bonfire/Mobile/Utilities/Observable+Extensions.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+import RxSwift
+
+func mergeToArray<T>(observables: [Observable<T>]) -> Observable<[T]> {
```

Comment:
I know I was pretty responsible for this name, but maybe we can be more accurate / descriptive?

`convertToObservableArray` maybe?

`convertArrayOfObservablesToObservableArray` too much?